src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -404,6 +405,11 @@ class FeatureSearchStore implements IFeatureSearchStore {
 
             if (!entry) {
                 // Create a new entry
+                const name =
+                    row.user_name ||
+                    row.user_username ||
+                    row.user_email ||
+                    'unknown';
```

Comment:
Is the unknown case where we don't know who created the flag initially? Like Ivar asked, what do we show then?

Reply:
We show this
<img width="143" alt="Screenshot 2024-06-06 at 09 42 21" src="https://github.com/Unleash/unleash/assets/1394682/573d5344-4bfd-4009-bb87-5db9b53aad8a">